src/main/java/org/broadinstitute/hellbender/tools/sv/Pedigree.java

```diff
@@ -0,0 +1,136 @@
+package org.broadinstitute.hellbender.tools.sv;
+
+import java.io.*;
+import java.util.*;
+
+public class Pedigree {
```

Comment:
Public classes need top level comments explaining what they are, especially important for things like this where you probably want to link to the file format you expect.  PedReader has very good example documentation.  (Probably more than really necessary for all classes, but helpful)